nyaa/templates/layout.html

```diff
@@ -48,6 +48,11 @@
 		<script src="{{ static_cachebuster('js/bootstrap-select.js') }}"></script>
 		<script src="{{ static_cachebuster('js/main.js') }}"></script>
 
+		<!-- Include Date Range Picker -->
+		<script type="text/javascript" src="//cdn.jsdelivr.net/momentjs/latest/moment.min.js"></script>
```

Comment:
please use cloudflare's CDNJS with an integrity check, no need to contact even more domains